### ISSUE TITLE
Prototype of org website design needed

### ISSUE THREAD

**Issue by kakirastern** [wildfirepy/wildfirepy.org#1]
We are in the process of planning for a new website for the `WildfirePy` project, and will need some prototype(s) of the design drafted. The idea is to have a theme ready within the next few weeks. We will need to have the following pages' design to start:

- [ ] Homepage (or landing page)
- [ ] Header
- [ ] Footer
- [ ] About page(s)
- [ ] Contact page
- [ ] Team page(s)
- [ ] Search page(s)
- [ ] Get Help page

**Comment by eshandhawan51** [comment]
I also thought we could some developers page
also add some links  to the various GitHub repos and communication channels 
what are your thought on it @kakirastern @Raahul-Singh

**Comment by kakirastern** [comment]
Yeah, sure. But should this section be under the Team's page? That way it would be better organized.

**Comment by eshandhawan51** [comment]
while I thought of keeping it as a tab on the main page 
this way it would be easily accessible to the new developers

**Comment by kakirastern** [comment]
Hi @eshandhawan51 How's the progress?

**Comment by eshandhawan51** [comment]
@kakirastern I was quite busy with my GSOC work 
I will look into it, as soon as I get some time :)
